examples/latex.jl

```diff
@@ -15,13 +14,14 @@ function main(window)
              textinput() >>> texᵗ,
              hbox("Show as a block", hskip(1em), checkbox(mode) >>> modeᵗ),
              vskip(1em),
-             tex(t, block=mode)) |> pad(1em) |> maxwidth(30em)
+             tex(t, block=mode)) |> Escher.pad(1em) |> maxwidth(30em)
```

Comment:
I think always qualifying is a good idea. I'm thinking we can make an alias named `padding` so this doesn't need to look ugly.